src/components/DayPickerRangeController.jsx

```diff
@@ -413,36 +418,49 @@ export default class DayPickerRangeController extends React.Component {
   }
 
   onDayClick(day, e) {
-    const { keepOpenOnDateSelect, minimumNights, onBlur } = this.props;
+    const {
+      keepOpenOnDateSelect, minimumNights, onBlur, range,
```

Comment:
if the entire statement can't fit on one line, each property needs to be on its own line.

Reply:
👍